src/System.Runtime/samples/DateTime-formatting/DateTime-formatting.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Globalization;
+
+public class Program
+{
+	public static void Main()
+	{
+		// Demonstrates common uses of formatting the DateTime struct.
+		// Doc: https://msdn.microsoft.com/library/az4se3k1.aspx
+		
+		var date = DateTime.Now;
+		
+		// default formatting
+		Console.WriteLine("Print DateTime with default formatting: {0}", date);
+		Console.WriteLine();
+		
+		// short / long formatting
+		Console.WriteLine("Print short date: {0}", date.ToString("d"));
+		Console.WriteLine("Print long date: {0}", date.ToString("D"));
+		Console.WriteLine("Print short time: {0}", date.ToString("t"));
+		Console.WriteLine("Print long time: {0}", date.ToString("T"));
+		Console.WriteLine();
```

Comment:
I think it would be better to teach people to do:

            Console.WriteLine("Print short date: {0:d}", date);
            Console.WriteLine("Print long date: {0:D}", date);
            Console.WriteLine("Print short time: {0:t}", date);
            Console.WriteLine("Print long time: {0:T}", date);

Not that it's important for console scenarios, but this form can be optimized to not allocate strings.